### PR TITLE
Add recsys part 2

### PR DESCRIPTION
**Pull request by lilianabs**
Adding part 2 for the recsys project. The scope has been reduced so we can concentrate on metrics and comparing the two recommender systems we created during part 1.

### CODE REVIEW

File: 2022-10-19-recommenders-1.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ Found good materials? Create a PR with links!
 * 🗒️ [Introduction to recommender systems](https://towardsdatascience.com/introduction-to-recommender-systems-6c66cf15ada)
 * 🗒️ [Recommender Systems: Machine Learning Metrics and Business Metrics](https://neptune.ai/blog/recommender-systems-metrics)
 * 🗒️ [Building Recommender Engines: Challenges and Opportunities](https://youtu.be/PjVEan8QgCI)
+* 💻 [Sample project](https://github.com/alexeygrigorev/product-recommendation) for Part 1.
```
Comment (alexeygrigorev) [inline 2022-10-19-recommenders-1.md]: Probably not needed? I added it to the list of projects at the bottom of the file  - maybe that should be sufficient?

File: 2022-11-16-recommenders-2.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+* Recap of the project you work on during Part 1 of this project-of-the-week.
+* If you couldn’t work on Part 1 of this project-of-the-week, then you can catch up by quickly reviewing some of the suggested materials.
+* Create a GitHub project
+* Share your progress in Slack and on social media
+
+Suggested materials
+
+* 💻 [Sample project](https://github.com/alexeygrigorev/product-recommendation) for Part 1.
+
+Would you like to add your project for Part 1? Create a PR with links!
```
Comment (alexeygrigorev) [inline 2022-11-16-recommenders-2.md:43]: Probably it makes sense to give a link to the part one?

File: 2022-11-16-recommenders-2.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+alternatives instead
+
+## Plan
+
+This is a proposed plan only, you don’t have to follow it day-by-day.
+
+
+### Day 0 (16 November, Wednesday)
+
+* Recap of the project you work on during Part 1 of this project-of-the-week.
```
Comment (alexeygrigorev) [inline 2022-11-16-recommenders-2.md]: can you please make "Part 1" linkable?

File: 2022-11-16-recommenders-2.md (hunk 2 above)
Comment (alexeygrigorev) [reply 2022-11-16-recommenders-2.md:43]: I mean turn it into a link =)

File: 2022-11-16-recommenders-2.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+Suggested materials
+
+* 💻 [Sample project](https://github.com/alexeygrigorev/product-recommendation) for Part 1.
+
+Would you like to add your project for Part 1? Create a PR with links!
+
+
+### Day 1 (17 November, Thursday)
+
+* Learn about evaluation metrics for recommender systems.
```
Comment (alexeygrigorev) [inline 2022-11-16-recommenders-2.md:48]: Maybe we can add a list of metrics we should study? Like for day 1 see which metrics are available and learn about something like average precision and MAP

What I also find helpful is already having some baseline solution (e.g. sorting by frequency) and trying to evaluate it.

File: 2022-11-16-recommenders-2.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+Suggested materials
+
+* 🗒️ [Evaluation Metrics for Recommender Systems](https://towardsdatascience.com/evaluation-metrics-for-recommender-systems-df56c6611093)
+* 🗒️ [Recommender Systems: Machine Learning Metrics and Business Metrics](https://neptune.ai/blog/recommender-systems-metrics)
+* 📺 [Week 6: Recommender Systems - Part 4: Evaluation of Recommender Systems](https://www.youtube.com/watch?v=mpv8iMe24-Q)
+
+
+### Day 2 (18 November, Friday)
+
+* Continue learning about evaluation metrics for recommender systems.
```
Comment (alexeygrigorev) [inline 2022-11-16-recommenders-2.md:65]: also probably it will help to have a suggested list with metrics

File: 2022-11-16-recommenders-2.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+* 🗒️ [Evaluation Metrics for Recommender Systems](https://towardsdatascience.com/evaluation-metrics-for-recommender-systems-df56c6611093)
+* 🗒️ [Recommender Systems: Machine Learning Metrics and Business Metrics](https://neptune.ai/blog/recommender-systems-metrics)
+* 📺 [Week 6: Recommender Systems - Part 4: Evaluation of Recommender Systems](https://www.youtube.com/watch?v=mpv8iMe24-Q)
+
+Found good materials? Create a PR with links!
+
+### Day 3 (19 November, Saturday)
+
+* Compare the recommender systems you created using the metrics that you selected.
```
Comment (alexeygrigorev) [inline 2022-11-16-recommenders-2.md]: recommender systems from week 1, right? the content based and collaborative filtering? 

Maybe add a note that you can take a look at projects from part one if you don't have the models

**Review by alexeygrigorev — COMMENTED** [review]
Looks really great, thank you for your work Liliana! A few changes and we can merge it

File: 2022-10-19-recommenders-1.md (hunk 1 above)
Comment (lilianabs) [reply 2022-10-19-recommenders-1.md]: Yeah, I think we don't need to repeat resources :)

File: 2022-11-16-recommenders-2.md (hunk 2 above)
Comment (lilianabs) [reply 2022-11-16-recommenders-2.md:43]: Great point! We definitely need to add a link to part 1.

File: 2022-11-16-recommenders-2.md (hunk 3 above)
Comment (lilianabs) [reply 2022-11-16-recommenders-2.md]: Sure!

File: 2022-11-16-recommenders-2.md (hunk 4 above)
Comment (lilianabs) [reply 2022-11-16-recommenders-2.md:48]: This is really helpful. I'm adding these points.

File: 2022-11-16-recommenders-2.md (hunk 5 above)
Comment (lilianabs) [reply 2022-11-16-recommenders-2.md:65]: I'm adding a list but please let me know if the list is ok.

File: 2022-11-16-recommenders-2.md (hunk 6 above)
Comment (lilianabs) [reply 2022-11-16-recommenders-2.md]: Adding these points :)

File: 2022-11-16-recommenders-2.md (hunk 5 above)
Comment (alexeygrigorev) [reply 2022-11-16-recommenders-2.md:65]: it's a bit long but it's okay!

**Comment by alexeygrigorev** [comment]
I think it's ready to be merged. Maybe let's wait a few more day - maybe Elias or Antonis have some feedback

File: 2022-11-16-recommenders-2.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+* Compare the recommender systems you created (Content based and Collaborative filtering) using the metrics that you selected.
+* Push your changes to GitHub.
+* Share your progress in Slack and on social media.
+
+**Note:** If you didn't have the chance to work on the project, have a look at projects from [Part 1](2022-10-19-recommenders-1.md).
+
+### Day 4 (20 November, Sunday)
+
+* Continue exploring more about this topic.
```
Comment (eliasnema) [inline 2022-11-16-recommenders-2.md:95]: Just an idea: should we ask people to write up the comparisons of their recsys algos using the metrics they learned above and come up with a conclusion?

**Comment by alexeygrigorev** [comment]
I'll merge it for now so I can include it in the newsletter tomorrow. But feel free to create a PR if you want to add anything

**Pull request merged**